src/hooks/useMemberListener.js

```diff
@@ -4,6 +4,18 @@ import useContract from "hooks/useContract";
 import { useMember } from "providers/MemberData";
 import { useVouchers } from "providers/VouchersData";
 
+// The useContract() calls in useMemberListener() return undefined
+// when no chain is connected causing the useContractEvent() hooks to fail.
+//
+// It looks like this behaviour is fixed in recent wagmi versions but
+// as a workaround until we upgrade this component allows us to conditionally
+// listen for the contract events.
+export const MemberListener = ({ children }) => {
```

Comment:
Hmm can we just return mainnet as the default for the useContract? Or does that break other stuff, would be worth trying that.

Reply:
I've refactored it. That should be fine as the contract event listener conditions will never pass while the `account` is undefined.